jwt.js

```diff
@@ -1,61 +1,106 @@
-'use strict'
-
 const fp = require('fastify-plugin')
 const JWT = require('jsonwebtoken')
 const assert = require('assert')
+const waterfall = require('async/waterfall')
```

Comment:
Can you use [`steed`](https://github.com/mcollina/steed) instead of `async`?

Reply:
I love how you guys have a module replacement for literally everything and they are also super fast 😄